### PR TITLE
chore: add tests, warning fixes

### PR DESCRIPTION
**Pull request by theAndrewCline**
add a snapshot test
add lodash to dependancies
add unique keys for todos

### CODE REVIEW

**Comment by theAndrewCline** [comment]
Hey Grant! Looks at dis code.

File: src/App.js [hunk 1]
```diff
@@ -60,7 +61,7 @@ function Todos(props) {
 	return (
 		<div>
 			{props.todos.map(todo => (
-				<Todo val={todo} deleteTodo={props.deleteTodo} />
+				<Todo val={todo} deleteTodo={props.deleteTodo} key={_.random}/>
```
Comment (GeeDollaHolla) [inline src/App.js]: Lodash! Are you kidding?!

Comment (theAndrewCline) [reply src/App.js]: @GeeDollaHolla Lodash is cool man.

**Comment by theAndrewCline** [comment]
Hi @GeeDollaHolla I updated the code with a new commit. I removed those pesky default list values

**Comment by GeeDollaHolla** [comment]
Looks good. Should help prevent any key conflicts

**Pull request merged**